packages/react/project.json

```diff
@@ -8,7 +8,6 @@
             "executor": "nx:run-commands",
             "options": {
                 "commands": [
-                    "tsc -b -w ./tsconfig.package.json",
```

Comment:
If I remember correctly, the only reason why the React package has a project.json is that we need a complicated command for React here.
We simplified the command for the React package.

Maybe we can delete project.json for this package now?